src/services/EnvConfigService.ts

```diff
@@ -0,0 +1,42 @@
+import { singleton, injectable } from "tsyringe";
+
+import { GroupMeInfo } from "./GroupMeService";
+
+@singleton()
+@injectable()
+export class EnvConfigService {
+  opts: EnvConfigServiceOpts;
+
+  public constructor(opts: EnvConfigServiceOpts) {
+    this.opts = opts;
+    console.log(`EnvConfigServiceOpts: ${JSON.stringify(this.opts)}`);
+  }
+
+  /**
+   * BotID returns Dev BOT_ID by default
+   * If PROD GroupMe ID is passed to CTOR, Prod BOT_ID is returned
+   */
+  public get BotID(): string {
+    return this.opts.groupID !== this.ProdGroupID ? this.ProdBotID : this.DevBotID;
```

Comment:
when ready to merge, this should be `return this.opts.groupID !== this.ProdGroupID ? this.ProdBotID : this.DevBotID;`

Reply:
fix ye?